plugins/modules/system/vdo.py

```diff
@@ -258,6 +258,15 @@
               configured setting unless a different value is specified
               in the playbook.
         type: str
+    force:
+        description:
+            - When creating a volume, ignores any existing file system
+              or VDO signature already present in the storage device.
+              When stopping or removing a VDO volume, first unmounts
+              the file system stored on the device if mounted.
```

Comment:
From what I understood, there should also be a warning that this can be destructive (with "use at your own risk"):
```suggestion
              the file system stored on the device if mounted.
            - "B(Warning:) <insert warning text here>."
```

Reply:
Sure, that sounds good, I'm gonna add a warning text later today.

Reply:
Would you please check again? https://github.com/ansible-collections/community.general/pull/2110/commits/5f4ab5928eba8e09d0846444e077f70b5147e8e3

Reply:
@rhawalsh Thanks for your suggestion, I have applied it in https://github.com/ansible-collections/community.general/pull/2110/commits/9033d385903e6fa4dd9f22f02eb73df929481d48.
Could you and @felixfontein please check this again?

Reply:
Looks good to me!